src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/query/FindProblemMessageTest.java

```diff
@@ -39,11 +40,11 @@ void setUp() {
     @Test
     void should_retrieve_an_ready_message() {
         List<String> caseEventMessages = findProblemMessageJob.run();
-        Assertions.assertThat(caseEventMessages.isEmpty()).isFalse();
-        Assertions.assertThat(caseEventMessages.size()).isEqualTo(3);
-        Assertions.assertThat(caseEventMessages.get(0)).isEqualTo("ID:c05439ca-ddb2-47d0-a0a6-ba9db76a3064:58:1:1-10");
-        Assertions.assertThat(caseEventMessages.get(1)).isEqualTo("ID:d257fa4f-73ad-4a82-a30e-9acc377f593d:1:1:1-2704");
-        Assertions.assertThat(caseEventMessages.get(2)).isEqualTo("ID:ce8467a0-cea9-4a65-99dd-3ae9a94a4453:16:1:1-811");
+        assertFalse(caseEventMessages.isEmpty());
```

Comment:
I know that this is picky but at SSCS I was told to use assertThat() over other forms of check.
"I would recommend using assertThat(result).isEqualTo(expectedResult) instead of assertEquals, btw, it's a lot more powerful and it helps avoid excessive imports"

Reply:
I have taken you comment onboard and made the relevant changes.